docs/game/screens/战斗画面.md

```
@@ -0,0 +1,89 @@
+
+# 战斗画面
+
+## 何时出现 + 状态流转
+
+进入任何副本/挑战后自动进入。退出条件：
+- 敌人全灭 → **战斗结果**子态（完成/再来一次/退出/撤退/已达成/倒带）
+- 按 esc / 点左上角 → **战斗-菜单**（暂停菜单）
+- 角色全灭 → 失败结算
+
+**子态**：
+| 子态 | 入口 | 出口 |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)